app/assets/stylesheets/forms.scss

```diff
@@ -0,0 +1,55 @@
+// .trans {
+//   background: rgba(0, 0, 0, 0.5);
+//   margin-top: 2rem;
+//   color: $white !important;
+//
+//   label {
+//     color: $white !important;
+//   }
+//
+//   .report-title h1 {
+//     color: $white !important;
+//   }
+// }
+
+// label {
+//   color: $white !important;
+// }
+
+.trans {
+  background: rgba(0, 0, 0, 0.5);
+  border-radius: 2px;
+  box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);
```

Comment:
Color literals like `rgba(0, 0, 0, 0.12)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `rgba(0, 0, 0, 0.24)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Commas in function arguments should be followed by a single space